.github/workflows/test.yml

```diff
@@ -4,17 +4,15 @@ jobs:
   test:
     strategy:
       matrix:
-        rust-toolchain: [stable]
+        rust-toolchain: [1.83.0]
         platform: [ubuntu-latest]
     runs-on: ${{ matrix.platform }}
     environment: test
     steps:
     - name: Checkout code
       uses: actions/checkout@v3
     - name: Install Rust
-      uses: dtolnay/rust-toolchain@stable
-      with:
-        toolchain: ${{ matrix.rust-toolchain }}
```

Comment:
If the matrix is not being used anymore then we should remove it from the top of the file entirely – having `rust-toolchain` configurable at the top but then not using that value could be confusing.

Reply:
Yeah, that was a larger change I wasn't ready to make on Friday... it was being used before, but only had one value in it, so the matrix was pointless. Over the next few weeks I plan to change all of these repositories to use a single reusable workflow so this stuff isn't duplicated in every starter kit repository, and I'll take care of this at that point.